com.gitlab.zehkira.Myuzi.yml

```diff
@@ -0,0 +1,23 @@
+app-id: com.gitlab.zehkira.Myuzi
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: myuzi
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-config/myuzi:create
```

Comment:
Any reason it uses hardcoded `~/.config` instead of following `XDG_CONFIG_HOME` environment variable?

Reply:
It does follow `XDG_CONFIG_HOME`.

Reply:
Then it doesn't need this line. XDG_CONFIG_HOME is readable by default.

Reply:
It didn't seem to work in the test build though.